cdci_data_analysis/analysis/parameters.py

```diff
@@ -448,13 +467,13 @@ def check_int_value(value, units=None,name=None):
               value=ast.literal_eval(value)
             except:
                 pass
-            value=np.int(value)
-            if type(value) == int or type(value) == np.int:
-                pass
-            elif type(value) == float or type(value) == np.float:
-                pass
-            else:
-                raise RuntimeError('type of ', name, 'not valid', type(value))
+            try:
+                # TODO a different and bigger precision (eg float64) should be considered?
+                if type(value) == float:
+                    logger.warning("the value %s will be casted to int" % value)
```

Comment:
maybe warning is not sufficient here. This potentially changes the number. I would raise something leading to invalid usage.
If someone passes "10.0" to max_poinings = it's their fault.